Set textdomain in blueman's binscripts

infirit opened the pull request:
I am confused translations worked at all :confused:. Here is what I see without this commit.
![afbeelding](https://user-images.githubusercontent.com/3465730/89089535-16db8480-d39e-11ea-8697-41c4cdf92ed6.png)

cschramm commented:
eb6492f1 broke it, specifically the explicit `from gettext import gettext as _` (in `ManagerMenu.py` in your case, also e. g. `ManagerDeviceMenu.py`).

There are gettext `install()` calls for the correct domain in `Constants.py` and `Sdp.py`. It seems that the explicit import beforehand somehow breaks them. :thinking:

cschramm commented:
Ah, sure, `GNUTranslations.install` adds a `_` to `builtins` that points to its own `gettext` method while `gettext.gettext` depends on the currently set textdomain to find the translations and if it gets imported as `_` it shadows the builtin. If we always use an import and not the builtin (and mypy should complain if we don't), I think there's no point in any `install()` call at all. 

While we have to handle `BLUEMAN_SOURCE` before any (possibly) local imports, I think we should move other boilerplate stuff like translations, logging, signal handling, proc title etc. to a common place.

infirit commented:
The minimal required is setting the textdomain in the binscripts so no need to bind or install, updated the PR.

Also dropped the install in Sdp and Constants as it does not make sense to me, if we always imported these in the binscripts maybe.
